tests/unit/amo/components/TestAddonMoreInfo.js

```diff
@@ -184,6 +185,20 @@ describe(__filename, () => {
     expect(root.find('.AddonMoreInfo-version').children()).toHaveText('2.0.1');
   });
 
+  it('renders file size of an add-on', () => {
+    _loadVersions({
+      files: [
+        {
+          ...fakePlatformFile,
+          size: 10,
+        },
+      ],
+    });
+    const root = render({});
+
+    expect(root.find('.AddonMoreInfo-filesize').children()).toHaveText('10 B');
```

Comment:
Could you please use the `formatFilesize` (from `core/i18n/utils`) in `toHaveText`?

```js
expect(root.find('.AddonMoreInfo-filesize').at(0)).toHaveText(formatFilesize(size));
```

In the snippet above, I also extract `size` as a variable so that you can use it in the `_loadVersions` call as well.